packages/ember-data/tests/integration/relationships/belongs-to-test.js

```diff
@@ -602,7 +602,7 @@ test("belongsTo hasData async loaded", function () {
 
   run(function() {
     store.find('book', 1).then(function(book) {
-      var relationship = book._relationships['author'];
+      var relationship = book._internalModel._relationships['author'];
```

Comment:
test reaching two levels into private API seems dubious, at the very least this should be factored into a test helper.